src/Icon/Icon.jsx

```diff
@@ -40,11 +40,9 @@ export const Icon = ({ name, size, color, rotate, ...props }) => {
   )
 }
 
-const BaseIcon = styled(DefaultIcon)(({ color }) =>
+const BaseIcon = styled(DefaultIcon)(({ svgColor }) =>
   css({
-    '& path': {
-      fill: color
-    }
+    color: svgColor
```

Comment:
Are you sure this works as expected? I know that:
- when using SVG with paths, we must use `fill` for the paths inside the SVG
- when using font icons, we must use `color`.
Are you sure this fix covers both cases? 

Also, why the prop rename?

Reply:
The base icon is the one from `@mdi/react` and by default the color prop is `currentcolor` 
Source: https://github.com/Templarian/MaterialDesign-React/blob/master/src/Icon.tsx#L16

If we use the color prop the path will be directly affected
```
 if (color !== null) {
    pathStyle.fill = color;
  }
```
Source: https://github.com/Templarian/MaterialDesign-React/blob/master/src/Icon.tsx#L46

My solution is that we don't change the default `color` prop and instead we use the `svgColor` to style the svg color with the `css()` so we can access the theme colors

Reply:
Is there a use case when we'll have `<Icon color='red' svgColor='disabled' />`? If not, I don't see a reason to keep two color props.

The underlying color extractor is able to make a difference between theme colors, literal colors and hex/rgb/rgba colors and properly handles them.